test/integration/lib/ipcamera_server.js

```diff
@@ -49,32 +49,37 @@ var ipCamera_server = (function() {
        '<presentationURL>http://' + ip + ':' + port + '</presentationURL>' +
        '</device>' +
        '</root>'
-      );
+       );
     });
-    
+
     _server.get('/image/jpeg.cgi', function (req, res) {
-        //send a jpeg
-        console.log('Camera: snapshot request received');
-        res.status(200).sendFile(
-          path.join(__dirname,'colville.jpg'));
+      //send a jpeg
+      console.log('Camera: snapshot request received');
+      res.status(200).sendFile(
+        path.join(__dirname,'colville.jpg'));
     });
 
-    instance = _server.listen(parseInt(port), function () {
+    return new Promise(resolve => {
+      // Stopping cameraUPnPServer causes socket error on next script. disabled.
+      // Also, Wireshark does not show additional traffic even when not stopped.
```

Comment:
I'm not sure what this comment is about when starting the server. Is it a copy and paste issue [from here](https://github.com/fxbox/foxbox/pull/481/files#diff-14e2cb55c60a4b51fa665e1ed344c25cR75)